elytron/src/main/java/org/wildfly/extension/elytron/PermissionMapperDefinitions.java

```diff
@@ -404,9 +403,8 @@ static Permissions createPermissions(List<Permission> permissionsList) throws St
     private static java.security.Permission createPermission(Permission permission) throws StartException {
         Module currentModule = Module.getCallerModule();
         if (permission.getModule() != null && currentModule != null) {
-            ModuleIdentifier mi = ModuleIdentifier.fromString(permission.getModule());
             try {
-                currentModule = currentModule.getModule(mi);
+                currentModule = currentModule.getModule(permission.getModule());
```

Comment:
Please update the Permission class constructor to call ModuleIdentifierUtil.canonicalModuleIdentifier(module).

Like the others, in the future I want to do the canonicalization right when the value is put into the configuration model. But until then I'd rather have it done as close as possible to that code, which is the Permission constructor.